src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/TransactionService.java

```diff
@@ -26,4 +26,17 @@ public Transaction getTransaction(String transactionId, String passthroughHeader
             throw new ServiceException("Error Retrieving Transaction " + transactionId, e);
         }
     }
+
+    public void updateTransaction(Transaction transaction, String passthroughHeader) throws ServiceException {
+        try {
+            var uri = "/private/transactions/" + transaction.getId();
```

Comment:
Ah, another URI. So yes, how about a URI constants class to collect all these up?